### PR TITLE
adWMG adapter: add new parameter, fix minor bugs

### PR DESCRIPTION
**Pull request by nyakove**
## Type of change

- [x] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Add new bid parameter - 'floorCPM'
Fix minor bugs in user sync

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by nyakove** [comment]
docs PR: https://github.com/prebid/prebid.github.io/pull/2683

File: modules/adWMGBidAdapter.js [hunk 1]
```diff
@@ -113,14 +113,18 @@ export const spec = {
     return bidResponses;
   },
   getUserSyncs: (syncOptions, serverResponses, gdprConsent, uspConsent) => {
-    if (gdprConsent) {
+    if (gdprConsent && !SYNC_ENDPOINT.includes('gdpr')) {
```
Comment (harpere) [inline modules/adWMGBidAdapter.js:116]: this is causing tests to fail because `.includes()` is not supported in `IE11`

**Review by harpere — CHANGES_REQUESTED** [review]
I'd recommend adding some tests to verify your changes.

**Comment by nyakove** [comment]
Hi @harpere! Thanks for review. I've made some changes (use _.indexOf_ instead of _.includes_ for compatibility with IE11) and add unit tests to cover code changes.

As about new param 'floorCPM', it is non-required and passes as is in _params_ object here:
`bids: {
          bidder: bidRequest.bidder,
          **params: bidRequest.params**
        },`
        
Because it's non-required, I do not check it in adapter's code

**Review by harpere — APPROVED** [review]

**Pull request merged**